packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -12,8 +12,11 @@ import { Stable } from '@agoric/internal/src/tokens.js';
 import { TimeMath } from '@agoric/time/src/timeMath.js';
 import { makePromiseKit } from '@endo/promise-kit';
 
-import { instanceNameFor } from './price-feed-proposal.js';
```

Comment:
yay for reducing coupling here!

These go into separate bundles, so including price-feed in addAssetToVault was awkward. This should reduce bundle sizes.